libs/libtirpc/patches/800-bzero-to-memset.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/des_impl.c
++++ b/src/des_impl.c
+@@ -588,7 +588,7 @@ _des_crypt (char *buf, unsigned len, str
+     }
+   tin0 = tin1 = tout0 = tout1 = xor0 = xor1 = 0;
+   tbuf[0] = tbuf[1] = 0;
+-  __bzero (schedule, sizeof (schedule));
++  memset (schedule, 0, sizeof (schedule));
```

Comment:
Thanks for the info, though I can't claim to understand the possible security issues. I do know that `__bzero` errors and fails to compile in my environment, after searching I found emails and code changes in the historu from `__bzero` to `memset` and so I assumed that this one had been missed and mimicked their other changes. If I were able to get the code you point to work then would I also have to go through the rest of the source and do he same with every `memset` that I can find? Perhaps better to propose it upstream?